src/main/kotlin/com/terraformation/backend/customer/api/OrganizationsController.kt

```diff
@@ -66,8 +70,10 @@ class OrganizationsController(
   ): ListOrganizationsResponsePayload {
     val elements =
         organizationStore.fetchAll(depth).map { model ->
-          OrganizationPayload(model, getRole(model))
+          OrganizationPayload(
+              model, getRole(model), userStore.getTerraformationContactUser(model.id))
```

Comment:
This will throw an exception if the user doesn't have permission to see the contact for an organization, which they won't if they have the Contributor role.

Reply:
Since this is only necessary to include for the API call that fires within the accelerator console side of Terraware, is it acceptable to swallow the error? Or should I do a more explicit "get tf contact user if user `canListOrganizationUsers`" check?

Reply:
I went with the latter